src/react/components/SvgRadioButton.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Select, Tooltip } from 'oskari-ui';
+import { Form, Card, Space, Input, Row, Radio } from 'antd';
+import styled from 'styled-components';
+
+const formLayout = {
+    labelCol: { span: 24 },
+    wrapperCol: { span: 24 }
+}
+
+const RadioIcon = styled(Radio.Button)`
+    height: 34px;    
```

Comment:
Feels like this should be given as a prop for the component if this is meant to be a generic component. You could just move this to a folder named after StyleForm and give it more specific name based on that.

Reply:
Size probably assumes something about the content of the options as well?